lib/vrl/parser/src/parser.lalrpop

```diff
@@ -40,6 +40,7 @@ extern {
         "?" => Token::Question,
         "|" => Token::Pipe,
```

Comment:
I think we can remove this (unused?) token, right?

Reply:
I recall trying, but think it is still used by the coalescing in paths.. I'll try again. It needs cleaning up regardless..